platform-includes/configuration/featureflags/javascript.angular.mdx

```diff
@@ -0,0 +1,17 @@
+```TypeScript
```

Comment:
this is generally lowercase, but I do see some instances where it is camel (might be good to do a refactor pr to make them all lowercase)

```suggestion
```typescript
```